tests/integration/tests/kiali_urls_test.go

```diff
@@ -16,3 +19,93 @@ func TestKialiStatus(t *testing.T) {
 	assert.True(response)
 	assert.Equal(200, statusCode)
 }
+
+func TestKialiConfig(t *testing.T) {
+	assert := assert.New(t)
+	response, statusCode, err := utils.KialiConfig()
+
+	assert.Nil(err)
+	assert.Equal(200, statusCode)
+	assert.NotEmpty(response)
+	assert.NotEmpty(response.ClusterInfo)
+	assert.NotEmpty(response.ClusterInfo.Name)
+}
+
+func TestIstioPermissions(t *testing.T) {
+	assert := assert.New(t)
+	response, statusCode, err := utils.IstioPermissions()
+
+	assert.Nil(err)
+	assert.Equal(200, statusCode)
+	assert.NotNil(response)
+}
+
+func TestJaeger(t *testing.T) {
+	if os.Getenv("TEST_JAEGER") != "true" {
+		return
+	}
```

Comment:
See if you can remove this after addressing the comment below. Having jaeger deployed and healthy should be a prerequisite of the integration tests.